src/components/CloudHeader/CloudHeader.js

```diff
@@ -0,0 +1,21 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import classNames from 'classnames';
+
+const CloudHeader = props => {
+  const { children, className, ...other } = props;
+
+  const cloudHeaderClasses = classNames('cloud-header', className);
+
+  return (
+    <nav className={cloudHeaderClasses} {...other}>
+      {children}
+    </nav>
+  );
+};
+
+CloudHeader.propTypes = {
+  children: PropTypes.node,
```

Comment:
Should we add a custom prop validator here to verify children are `CloudHeaderContainer`?